templates/paloalto_panos_show_interface_management.textfsm

```diff
@@ -0,0 +1,18 @@
+Value Name (\S+)
+Value MACaddress ([a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2}\:[a-fA-F0-9]{2})
+Value IPv4 (\S+)
+Value IPv4Netmask (\S+)
+Value IPv4Gateway (\S+)
+Value IPv6 (\S+)
+Value IPv6LinkLocal (\S+)
```

Comment:
Do you mind adding the necessary tests for this?

Reply:
Yes, see https://github.com/networktocode/ntc-templates/pull/681